docs-internal/api/nu-designer-openapi.yaml

```diff
@@ -345,12 +345,12 @@ paths:
                         process-version: '0.1'
                         engine-version: '0.2'
                         generation-time: '2023-09-25T09:26:30.402299'
-  /api/app/healthCheck/process/deployment:
```

Comment:
does it mean there was some problem with open API generation?

Reply:
I don't think so. Only the order has been changed. Or maybe you see some other problem?